pkg/restore/restic_restore_action.go

```diff
@@ -106,14 +106,22 @@ func (a *ResticRestoreAction) Execute(input *velero.RestoreItemActionExecuteInpu
 
 	cpuRequest, memRequest := getResourceRequests(log, config)
 	cpuLimit, memLimit := getResourceLimits(log, config)
+	if cpuRequest == "" {
+		cpuRequest = defaultCPURequestLimit
+	}
+	if cpuLimit == "" {
+		cpuLimit = defaultCPURequestLimit
+	}
+	if memRequest == "" {
+		memRequest = defaultMemRequestLimit
+	}
+	if memLimit == "" {
+		memLimit = defaultMemRequestLimit
+	}
 
 	resourceReqs, err := kube.ParseResourceRequirements(cpuRequest, memRequest, cpuLimit, memLimit)
 	if err != nil {
-		log.Errorf("Using default resource values, couldn't parse resource requirements: %s.", err)
-		resourceReqs, _ = kube.ParseResourceRequirements(
-			defaultCPURequestLimit, defaultMemRequestLimit, // requests
-			defaultCPURequestLimit, defaultMemRequestLimit, // limits
-		)
+		return nil, errors.Wrapf(err, "couldn't parse resource requirements: %s", err)
```

Comment:
I would probably revert this section to the way it was before, i.e. error-log and use the default values, as it's a little more forgiving to the user. Otherwise, LGTM!

Reply:
Revert for favor the user.